docs/components/App.js

```diff
@@ -41,7 +42,9 @@ export default function App({ children }: Props): Node {
       window.gtag('config', 'UA-12967896-44', {
         page_path: window.location.pathname + window.location.search + window.location.hash,
       });
+      setIsHomePage(window?.location?.pathname === '/home');
```

Comment:
If we're only showing the resources footer on the home page, does it make sense to move it into that page specifically? Rather than include it in AppLayout?

Reply:
Because fo the way our pages are styled, it involves a lot of extra styling to get the footer to display properly if I put it within the Home page content (it won't be full width or flush onto the other footer), so it felt like this was a better fit for how we approach laying out all these pages